packages/rspack/tests/statsCases/limit-chunk-count-plugin/webpack.config.js

```diff
@@ -15,8 +15,8 @@ module.exports = [1, 2, 3, 4].map(n => ({
 		})
 	],
 	stats: {
-		hash: false,
-		timings: false,
```

Comment:
Add this back to keep snapshot stable

Reply:
this is not included in original webpack tests, so we may don't need this